### PR TITLE
Update projects using Tether list

### PR DESCRIPTION
**Pull request by JKillian**
I made three separate (but very small) changes here:

* Added Blueprint
* Alphabetized non-hubspot projects
* Capitalized Ember Tether (now uses the same capitalization as its README)

### CODE REVIEW

**Comment by TrevorBurnham** [comment]
Thanks!

File: docs/2-Examples/2-projects_using_tether.md [hunk 1]
```diff
@@ -24,17 +24,21 @@ in making something which pops up next to something the user clicks or hovers on
...
+### [Bootstrap 4](http://getbootstrap.com/)
+
+Tooltip and popovers from Bootstrap 4 are positioned using Tether
 
-### [ember-tether](https://github.com/yapplabs/ember-tether)
+### [Ember Tether](https://github.com/yapplabs/ember-tether)
 
 An Ember.js-friendly interface for tether.
 
-### [Bootstrap 4](http://getbootstrap.com/)
```
Comment (svenluijten) [inline docs/2-Examples/2-projects_using_tether.md]: Why'd you remove Bootstrap here? AFAIK the alpha releases of Bootstrap 4 still use Tether for their dropdowns.

Comment (JKillian) [inline docs/2-Examples/2-projects_using_tether.md]: I didn't! I just alphabetized it because it seemed like this list was a little bit disorganized

Comment (svenluijten) [inline docs/2-Examples/2-projects_using_tether.md]: Oh I see. The diff I saw just showed the removal of Bootstrap. Carry on! 😄

**Pull request merged**